1-js/01-getting-started/3-code-editors/article.md

```diff
@@ -13,7 +13,7 @@ An IDE loads the project (which can be many files), allows navigation between fi
 If you haven't selected an IDE yet, consider the following options:
 
 - [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [WebStorm](https://www.jetbrains.com/webstorm/) (cross-platform, paid).
+- [WebStorm](https://www.jetbrains.com/webstorm/) (cross-platform, free for non-commercial use).
```

Comment:
@iliakan I was actually quite for this change as well. It could be "paid - free for non-commercial use"